backend/setupWebSecurity.js

```diff
@@ -11,16 +11,15 @@ const sixtyDaysInSeconds = 5184000
 const router = express.Router()
 
 module.exports = () => {
+  router.use(helmet())
```

Comment:
By default, helmet applies some basic hsts options but we supply our own on line 16.